### PR TITLE
migrate deleteEpoch FE to hasura

### LINKED ISSUE #639: Integrate Hasura Delete Epoch into FE
## Summary:
Integrate Hasura Delete Epoch into FE.

### Details
This task is to integrate the frontend of `<Delete Epoch>` to Hasura.
https://app.coordinape.com/admin

**Screenshot of Page**
image

**Logic:**
-

**Related Backend Issue:**
#565

### PR DESCRIPTION
**Pull request by CryptoGraffe**
Fixes #639 by migrating to hasura. 

Test plan:

1. Visit the admin page of a circle with an active epoch. In the fresh seed data Games in Collier and Sons. 
2. click the red trash next to the epoch on the top
3. click remove epoch button the dialog
4. observe that the epoch is deleted, but you have encountered bug #659 which is that the dialog does not dismiss. 
5. refresh the page to workaround #659

### CODE REVIEW

**Comment by CryptoGraffe** [comment]
This was working but is now blocked by #696

**Comment by CryptoGraffe** [comment]
> This was working but is now blocked by #696

Unblocked now.

**Comment by CryptoGraffe** [comment]
@zashton this needs re-approval after conflict resolution

**Review by zashton — APPROVED** [review]
tested and working

**Pull request merged**
